_docs/tasks/traffic-management/ingress.md

```diff
@@ -269,6 +269,344 @@ rules.
    curl -I -k https://$INGRESS_HOST/status/200
    ```
 
+## Configuring RBAC for ingress key/cert
+There are service accounts which can access this ingress key/cert, and the ingress key/cert is not 
+encrypted. This leads to risks of leaking key/cert. We can set up Role-Based Access Control ("RBAC") 
```

Comment:
This is not accurate. K8s supports encrypted secret already, see here: https://kubernetes.io/docs/tasks/administer-cluster/encrypt-data/ . It would be great if we could update this as well. But maybe in a follow up PR.

Reply:
Thanks for pointing that out. I didn't know that we can encrypt secret. I will document encrypting secrets in a follow up PR.